apps/etterlatte-saksbehandling-ui/client/src/components/aktivitetsplikt/vurdering/unntak/UnntakAktivitetspliktOppgave.tsx

```diff
@@ -118,7 +118,7 @@ export function UnntakAktivitetspliktOppgave({ formPrefix = '' }: { formPrefix?:
           </option>
         </Select>
 
-        <Textarea {...register(`${formPrefix}beskrivelse`)} label="Beskrivelse" />
+        <Textarea {...register(`${formPrefix}beskrivelse`)} label="Vurdering av unntak" />
```

Comment:
Burde ikke data'en få et likt navn?

Reply:
Tenker ikke det, er unødvendig å endre alle datafelter for en visningsdetalj.